runtime/syntax/vimnormal.vim

```diff
@@ -0,0 +1,16 @@
+syn match normalOp /[dcrypoaxv!"#%&.-\/:<>=?@ABCDGHIJKLMNOPQRSUVWXYZgmqstz~iu]/ nextgroup=normalMod
```

Comment:
Sorry, I forgot about this entirely. I think embeds are cleaner this way, but I'll open an issue upstream to see if we can take a different approach.